functions/jira/callAPI.ts

```diff
@@ -47,3 +55,34 @@ export function callAPI(
 
   return result;
 }
+
+interface IJiraSite {
+  id: string;
+  name: string;
+  url: string;
+  scopes: string[];
+  avatarUrl: string;
+}
+
+export async function getCloudID(
+  oAuthToken: string,
+  domainName: string,
+): Promise<string | undefined> {
+  let response = await callAPI(
+    baseOAuthURL,
+    API_OPS.getAccessibleResources(),
+    HEADERS.authHeader(oAuthToken),
+  );
+  console.log(response);
+  // @ts-ignore
```

Comment:
We want to avoid ignoring Typescript rules, was there no other way to solve this without disabling them?